ilc/client/CssTrackedApp.js

```diff
@@ -26,6 +26,7 @@ export class CssTrackedApp {
             mount: this.mount,
             unmount: this.unmount,
             update: this.update,
+            __CSS_TRACKED_APP__: true,
```

Comment:
Could you pls explain logic behind it ?

Reply:
Property added for unit test and easier debugging.